docs/source/articles/add-catalog-customizations.html.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Add Catalog Customizations
+excerpt: This guide shows you how to implement user-defined customizations for the products in your catalog.
+---
+
+# Add Catalog Customizations
+
+Customizations are pre-defined fields in your product that users can edit and create custom versions of your projects, such as for custom engravings.
+
+## Step 1: Create a `Catalog::Customizations` subclass
+
+The first step is to subclass `Workarea::Catalog::Customizations` with something descriptive of your customization. In this case, we'll be using the concept of an "engraving" for jewelry or other metal products:
+
+```ruby
+module Workarea
+  module Catalog
+    class Customizations::Engraving < Customizations
+      customized_fields :initials
+      validates_presence_of :initials
```

Comment:
```suggestion
      validates_presence_of :initials # this field is required
```

Reply:
i mean, kinda thought "validates presence of initials" spelled it out for ya?

Reply:
not sure what I was thinking... 🥇